pkg/start/start.go

```diff
@@ -240,7 +240,11 @@ func watchHostAgentEvents(ctx context.Context, inst *store.Instance, haStdoutPat
 				return true
 			}
 
-			logrus.Infof("READY. Run `%s` to open the shell.", LimactlShellCmd(inst.Name))
+			if *inst.Config.Plain {
+				logrus.Infof("READY. Run `ssh -F %q lima-%s` to open the shell.", inst.SSHConfigFile, inst.Name)
```

Comment:
Y not use LimactlShellCmd(inst.Name) here ?? Ssh connection stays the same right evn for plan mode ?

Reply:
Because plain mode users are less likely to prefer lima-ist commands.
The limactl shell command continue to work.

Reply:
I still think the `lima` wrapper (but only that script, not the others) belongs to the "minimal" footprint.

* https://github.com/lima-vm/lima/pull/1673

But I suppose `ssh` is even more "plain".